webpack/section.jsx

```diff
@@ -140,6 +159,10 @@ export default class App extends Component {
                   totalDuration={this.props.videoDuration}
                 />
               </div>
+              <SectionControls
```

Comment:
It would be nice if the shodan timeline maintained its collapsed/uncollapsed state when clicking between sections, but that's beyond the scope of this feature.

Reply:
Agreed. This could fold in with the `sessionStorage` still that @simonwiles has been working on.